src/inc/msquic.hpp

```diff
@@ -598,6 +600,15 @@ class MsQuicConfiguration {
                 sizeof(*QSettings),
                 QSettings);
     }
+
+    QUIC_STATUS
+    SetVersionNegotiationExtEnabled(_In_ const BOOLEAN Value) noexcept {
```

Comment:
Should be wrapped in `#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES`.

Reply:
What's that on line 591? 😄

Reply:
🤦🏻